services/src/main/scala/org/apache/gearpump/cluster/main/Services.scala

```diff
@@ -37,18 +38,58 @@ object Services extends App with ArgumentsParser {
   }
 
   override val options: Array[(String, CLIOption[Any])] = Array(
-    "master" -> CLIOption[String]("<host1:port1,host2:port2,host3:port3>", required = true))
+    "master" -> CLIOption[String]("<host1:port1,host2:port2,host3:port3>", required = true),
```

Comment:
Same story. You can use System.setProperty() to add that config, the properties will get loaded in Service automatically.